app/liquid_tags/user_subscription_tag.rb

```diff
@@ -73,12 +81,20 @@ class UserSubscriptionTag < LiquidTagBase
     }
 
     function showResponseMessage(noticeType, msg) {
-      clearSubscriptionArea();
+      clearSubscriptionArea(clearSubscribeButton = false);
+
       const responseMessage = document.getElementById('response-message');
       if (responseMessage) {
         responseMessage.classList.remove("hidden");
         responseMessage.classList.add(`crayons-notice--${noticeType}`);
         responseMessage.textContent = msg;
+
+        if (noticeType === 'danger') {
+          // Allow users to try resubscribing if they see an error message.
+          const subscribeBtn = document.getElementById('subscribe-btn');
```

Comment:
suggestion (non-blocking): Consider getting a reference to the subscribe button only one.

There's four spots where `const subscribeBtn = document.getElementById('subscribe-btn')` is called. I'm assuming it's the same reference, so in that case, you could just define it once. If it's a new reference because DOM is destroyed, this is fine.

Reply:
@nickytonline Great suggestion, as always! I've pulled that element out into a const.

I also added some margin-bottom padding to the logged-in-text so that the error doesn't look as pushed-up against the text whenever it renders:
<img width="703" alt="Screen Shot 2020-12-03 at 12 11 19 PM" src="https://user-images.githubusercontent.com/6921610/101082811-eb8dbf80-3560-11eb-9715-c76c88541400.png">